rtc/jwt-authentication.md

```diff
@@ -0,0 +1,157 @@
+---
+layout: default
+title: RTC JWT Authentication
+title_nav: JWT Authentication
+description: Guide on how to setup JWT Authentication for RTC
+keywords: jwt authentication
+---
+
+{% assign pluginname = "RTC" %}
+{% assign plugincode = "rtc" %}
+## Introduction
+
+RTC requires you to setup JSON Web Token (JWT) authentication. This is to ensure that only your authenticated users will be able to access and collaborate on documents.
```

Comment:
```suggestion
RTC requires setting up JSON Web Token (JWT) authentication. This is to ensure that only authenticated users will be able to access and collaborate on documents.
```
Avoiding `you` and `your` again.

Reply:
resolved in https://github.com/tinymce/tinymce-docs/pull/1778/commits/9cc200b9775691b5106c568717f3069b51dbb46e